pennylane/tape/transforms/metric_tensor.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Contains the metric tensor tape transform
+"""
+import numpy as np
+import pennylane as qml
+
+
+def _stopping_critera(obj):
+    if getattr(obj, "num_params", 0) == 0:
+        return True
+
+    if obj.name in ["RX", "RY", "RZ", "PhaseShift"]:
+        return True
+
+    return False

```

Comment:
Mabye just use a `Rot` in the test to cover this line?

Reply:
Good idea!